pkg/crawlerdb/db.go

```diff
@@ -1,23 +1,29 @@
-package main
+package crawlerdb
 
 import (
 	"bytes"
 	"database/sql"
 	"fmt"
 	"time"
 
-	_ "github.com/mattn/go-sqlite3"
+	_ "modernc.org/sqlite"
```

Comment:
The new library is essentially a project where they have converted the C library into a pure Go one. This is so that we don't need the requirement for CGO. Just one less thing. The source code is here: https://gitlab.com/cznic/sqlite